packages/paste-core/components/button/src/styles.ts

```diff
@@ -46,6 +45,7 @@ const baseButtonWrapper = (props: ButtonWrapperProps): SerializedStyles => css`
   background: none;
   transition: background-color 100ms ease-in, border-color 100ms ease-in;
   font-family: ${themeGet('fonts.fontFamilyText')(props)};
+  font-weight: ${themeGet('fontWeights.fontWeightSemibold')(props)};
```

Comment:
Even link style buttons should be semi bold and they are reset size by default https://share.goabstract.com/5343d021-abae-4f49-898e-c95df03667df?mode=design&sha=f133b3b4b51b7372f5805ed57bea98fce5e78d8d